contracts/anchor/Cargo.toml

```diff
@@ -39,8 +39,8 @@ ark-std = { version = "^0.3.0", default-features = false }
 ark-bn254 = { version = "^0.3.0", default-features = false, features = [ "curve" ] }
 ark-crypto-primitives = { version = "^0.3.0", features = ["r1cs"], default-features = false }
 
-arkworks-setups = { version = "0.5.3", default-features = false }
-arkworks-native-gadgets = { version = "0.5.3", default-features = false }
+arkworks-setups = { version = "1.0.0", features = ["r1cs"], default-features = false }
```

Comment:
Why not update to 1.1.1?

Reply:
Yes, when using the `arkworks-setups: 1.1.1`, the unit tests of `vanchor 16 * 2` are failing.
Could not find the special reason for that as well.